### PR TITLE
Update packages for release 0.2.0

### PR DESCRIPTION
**Pull request by ncclementi**
This PR updates the packages for the 0.2.0 release. I was able to conda-build locally, as well as install all the updated packages with a conda env. 

After this PR is merged, I will proceed with the release by bumping the version in the `setup.py` and the following steps for both pypi and conda-forge. 

For the regressions discussions check https://github.com/coiled/coiled-runtime/issues/526

cc: @hayesgb

### CODE REVIEW

**Comment by ncclementi** [comment]
The failure on the pip build is because there was a release of coiled that is already up on pypi but not on conda-forge yet. I would ignore it for the time being. 

```
E       AssertionError: assert <Version('0.2.51')> == <Version('0.2.50')>

tests/runtime/test_build.py:115: AssertionError
```

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {% set name = "coiled-runtime" %}
-{% set version = "0.2.0" + environ.get("VERSION_SUFFIX", '') %}
-{% set dask_version = environ.get("DASK_VERSION", "2022.6.0") %}
-{% set distributed_version = environ.get("DISTRIBUTED_VERSION", "2022.6.0") %}
+{% set version = "0.3.0" + environ.get("VERSION_SUFFIX", '') %}
```
Comment (phobson) [inline recipe/meta.yaml:2]: Are we releasing 0.2.0 or 0.3.0?

Comment (ncclementi) [reply recipe/meta.yaml:2]: This is for the nightlies, we usually put it one more up than the release it'll happen.

**Review by hayesgb — APPROVED** [review]
LGTM.  Based on conversation here and in #526 I would move forward.

**Pull request merged**